src/inc/til/env.h

```diff
@@ -231,41 +172,6 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
             }
         }
 
-        void get_computer_name()
-        {
-            if (auto value = til::details::wil_env::TryGetComputerNameW<std::wstring>(); !value.empty())
-            {
-                save_to_map(std::wstring{ til::details::vars::computer_name }, std::move(value));
-            }
-        }
-
-        void get_user_name_and_domain()
-        try
-        {
-            const auto token = wil::open_current_access_token();
-            const auto user = wil::get_token_information<TOKEN_USER>(token.get());
-
-            DWORD accountNameSize = 0, userDomainSize = 0;
-            SID_NAME_USE sidNameUse;
-            SetLastError(ERROR_SUCCESS);
-            if (LookupAccountSidW(nullptr, user.get()->User.Sid, nullptr, &accountNameSize, nullptr, &userDomainSize, &sidNameUse) || GetLastError() == ERROR_INSUFFICIENT_BUFFER)
-            {
-                std::wstring accountName, userDomain;
-                accountName.resize(accountNameSize);
-                userDomain.resize(userDomainSize);
-
-                SetLastError(ERROR_SUCCESS);
-                if (LookupAccountSidW(nullptr, user.get()->User.Sid, accountName.data(), &accountNameSize, userDomain.data(), &userDomainSize, &sidNameUse))
-                {
-                    strip_trailing_null(accountName);
-                    strip_trailing_null(userDomain);

```

Comment:
My intention was to just remove this code, but then I realized that it doesn't actually seem to do anything, because `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Environment` contains a `USERNAME` value (with the value-value of `SYSTEM`) which immediately overwrites this. Reading more into volatile environment variables, I've realized that this code can be replaced with one that simply copies them from the stale (process) environment block, because they are immutable across a login.

Reply:
clever. this means it is no longer a clone of `RegenerateUserEnvironment`, but that's OK.